pages/overview/index.mdx

```diff
@@ -16,6 +16,15 @@ for all supported packages!
 
 ## What is Auth.js?
 
+<Callout type="info">
+  Note that because of preventing breaking changes on package imports,
+  `next-auth` is the only framework package which is not named following the
+  `@auth/*` convention. This library initially was born as `next-auth` but [over
+  the years has evolved](/overview/contributors#history) to be framework
+  agnostic. That means Next.js will continue to install `next-auth`, while other
+  frameworks their `@auth/[framework]` variant
+</Callout>
```

Comment:
```suggestion
<Callout type="info">
  Note that because of preventing breaking changes on package imports,
  `next-auth` is the only framework package that is not named following the
  `@auth/*` convention. Auth.js was born out of `next-auth` but [over
  the years has evolved](/overview/contributors#history) to be framework
  agnostic. That means Next.js users will continue to install `next-auth` for now, while other
  frameworks their are available at `@auth/[framework]`.
</Callout>
```